composer.json

```diff
@@ -38,7 +38,7 @@
       "vendor/yiisoft/yii2/Yii.php"
     ],
     "psr-4": {
-      "Sil\\SspMfa\\Behat\\": "features/"
+      "SilInternational\\SspBase\\Features\\": "features/"
```

Comment:
If you want to keep it shorter, I think we've sometimes/often been using `Sil\` as the root of our class namespaces, rather than `SilInternational\`. Just a option. As long as what's here matches the `namespace` line in the files under "features/", I think it will work just fine (but you probably know better than I do whether that's correct).